app/controllers/provider_interface/application_data_export_controller.rb

```diff
@@ -35,8 +37,12 @@ def export
           .where(status: statuses)
           .where('candidates.hide_in_reporting': false)
```

Comment:
@mnacos added in https://github.com/DFE-Digital/apply-for-teacher-training/pull/5425/commits/97cf60292b15a7f884356a23542ec3fd222a5186 load tests indicate we get slightly better response times with batch sizes of `300`, db load is marginally higher than with initial test value of `150`. I've added as a _per report_ constant in case we want to tune this, it could just be a common value though?

Reply:
This is great, I was expecting `.find_each` to be marginally slower because of the gradual fetching. But you seem to have found a sweet spot wrt the interplay between streaming and db batching. I'd prefer using `.find_each` on principle, even if it was marginally slower!

Reply:
I think `per report` but set to the same value for now is 😋